_posts/newsletters/2020-04-13-newsletter.md

```diff
@@ -0,0 +1,77 @@
+
+In this issue:
+
+*   [US-RSE Virtual Workshop April 22-23](#virtual)
+*   [US-RSE Association Workshop at Princeton moving to October](#princeton)
+*   [Workshops at PEARC20 and SC20](#workshops)
+*   [Recent Job Postings](#jobs)
+*   [Interesting Reads](#reads)
+*   [Get Involved](#getinvolved)
+
+<a name="virtual"></a>
+# US-RSE Virtual Workshop
+
+We are excited to announce that the US-RSE association will host a virtual workshop **on April 22 and 23!** This workshop will be held in three online sessions:
+
+1.  The RSE Landscape - Reports on RSE groups and activities, with discussion (Wednesday 4/22, 12:00-1:30 PM EDT)
+2.  Technical talks - Short talks about projects that RSEs are working on (Wednesday 4/22, 3:00-4:30 PM EDT)
+3.  Next steps for the US-RSE organization - Breakout sessions to discuss ideas. What do *you* want from US-RSE? (Thursday 4/23, 12:00-1:30 PM EDT)
+
+A detailed agenda is in the works and will be posted online when ready.
+
+The workshop is free, but we ask that all participants preregister so that we know how many to expect and can prepare appropriately. **The registration form is** [**here**](https://docs.google.com/forms/d/e/1FAIpQLSeu6R65AZ_P5dQ0O-DTWn3k3a8E99AiqZX7PzKxe6X7t2a0AQ/viewform).
+
+The sessions will be held using Zoom. Connection information will be sent to registered participants before the workshop. (We've seen the recent news about Zoom security issues, and we will be taking appropriate security precautions.)
+
+If you have questions or would like to contribute a talk to one of the sessions, please contact [contact@us-rse.org](mailto:contact@us-rse.org) or reach out via slack. 
+
+<a name="princeton"></a>
+# US-RSE Association Workshop at Princeton is moving to October
+
+The in-person [US-RSE Community Building Workshop](https://us-rse.org/first-community-workshop/) has been officially rescheduled for October 27-28, 2020. It will be held at the same venue, the Nassau Inn in Princeton, NJ.
+
+This is a rapidly changing situation and it is impossible to predict what the travel/conference landscape will look like in October. Should COVID-19 continue to prevent us from having the workshop in October, we may need to reschedule for sometime in the following 12 months. Though we certainly hope it doesn't come to that.
+
+Follow the [US-RSE Event Page](https://us-rse.org/events-training/) to stay up to date.
+
+<a name="workshops"></a>
+# Workshops at PEARC20 and SC20
+
+We are thrilled to announce that in addition to the [PEARC20](https://us-rse.org/events/2020/2020-07-pearc20) workshop, we will have a workshop at [SC20](https://sc20.supercomputing.org/)! The PEARC20 workshop will be a Research Software Engineers Community Workshop, and the SC20 workshop will be "Research Software Engineers in HPC: Creating Community, Building Careers, Addressing Challenges (RSE-HPC-2020)." Stay tuned for more information.
```

Comment:
Should there be a note here as well about warranting these conferences are still on?